docs/11-dev-guide.md

```diff
@@ -11,7 +11,10 @@ copy that you built yourself.
 
 ### Install Prerequisite Software
 
-You will need the following software:
+Information about the prerequesite software and how to get an environment with these prerequesites already installed can
+be found in the [deployment guide](02-deployment-guide.md#install-prerequisite-software)
```

Comment:
This paragraph looks incorrect? The information for how to install the prerequisites is in this section.